src/stores/messages/MessagesStore.ts

```diff
@@ -153,7 +153,8 @@ export default class MessagesStore {
 	public scrollToMessage = (messageId: string) => {
 		const messageIndex = this.dataStore.messages.findIndex(m => m.messageId === messageId);
 
-		if (messageIndex === -1) throw new Error(`Message with ${messageId} id doesn't exists`);
+		if (messageIndex === -1 && this.dataStore.messages.length > 1)
```

Comment:
we can actually delete `scrollToMessage` and scrolledIndex

messageList scroll to message when `messagesStore.selectedMessageId` is changed

in MessagesUpdateStore replace `scrollToMessage` with assinging `messagesStore.selectedMessageId` new value